src/endpoints/admin.rs

```diff
@@ -67,7 +76,7 @@ pub async fn post_admin(
     }
 
     let pastas = {
-        let mut pastas = data.pastas.lock().unwrap();
```

Comment:
might want to swap the `?` for an `expect`

Std's mutex only returns an error if the lock was unlocked in another thread and that thread panicked. I think we should stop the program if any other thread panics. We can easily do so by panicking as well using a `.expect("no microbin thread should panick")`. The expect message is the "proof" why we should never have an error here.